### ISSUE TITLE
Django 1.6 & Python 3.3 installation error

### ISSUE THREAD

**Issue by ghost** [riklaunim/django-ckeditor#5]
``````
$ pip install django-ckeditor-updated
Downloading/unpacking django-ckeditor-updated
  Downloading django-ckeditor-updated-4.2.2.tar.gz (1.1MB): 1.1MB downloaded
  Running setup.py egg_info for package django-ckeditor-updated

    warning: no files found matching '*.asp' under directory 'ckeditor/static'
    warning: no files found matching '*.php' under directory 'ckeditor/static'
    warning: no previously-included files matching '*' found under directory 'ckeditor/static/ckeditor/ckeditor/_samples'
Requirement already satisfied (use --upgrade to upgrade): Pillow in /Users/lorddaedra/Envs/expo3/lib/python3.3/site-packages (from django-ckeditor-updated)
Installing collected packages: django-ckeditor-updated
  Running setup.py install for django-ckeditor-updated

    warning: no files found matching '*.asp' under directory 'ckeditor/static'
    warning: no files found matching '*.php' under directory 'ckeditor/static'
    warning: no previously-included files matching '*' found under directory 'ckeditor/static/ckeditor/ckeditor/_samples'
      File "/Users/lorddaedra/Envs/expo3/lib/python3.3/site-packages/ckeditor/management/commands/generateckeditorthumbnails.py", line 16
        print "Creating thumbnail for %s" % image
                                        ^
    SyntaxError: invalid syntax

Successfully installed django-ckeditor-updated
Cleaning up...```
``````

**Comment by ghost** [comment]
Please, update PyPi

**Comment by riklaunim** [comment]
There is another pull request on that Py3.3 issue. I need to merge that, do some more refactors and then release it with proper Py2.7/3.3 support. If you need it fast it could be done today/tomorrow.

**Comment by ghost** [comment]
Okay, I'll wait, thank you.

**Comment by riklaunim** [comment]
Fixed in 4.2.3
